kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/storage/PartitionLog.java

```diff
@@ -858,6 +864,37 @@ private void publishMessages(final Optional<PersistentTopic> persistentTopicOpt,
         });
     }
 
+    protected void checkAndRecordPublishQuota(Topic topic, int msgSize, int numMessages,
+                                              AppendRecordsContext appendRecordsContext) {
+        boolean isPublishRateExceeded;
```

Comment:
```suggestion
        final boolean isPublishRateExceeded;
```

Avoid the variable is not initialized.